### ISSUE TITLE
Tell Devel::Cover to ignore files outside current build directory

### ISSUE THREAD

**Issue by zmughal** [PDLPorters/devops#30]
This issue comes up in the coverage results of Test-PDL:

> -pdl-commits:#pdl- [Test-PDL] [build/push]: ∀S ✓ a8ba9e2 | Zaki Mughal [sivoais] | Use shared GitHub Actions from PDLPorters (#6)
>  https://github.com/ebaudrez/Test-PDL/actions/runs/1557475038
> [@sivoais] one odd thing with the coverage above: it thinks that a file from the PDL distribution is part of the coverage
> [@sivoais] a file that is located in ~/.cpanm ??
> [@sivoais] i saw this locally as well, but didn't bother tracking it down in Devel::Cover
> [@sivoais] so even though Test-PDL's coverage is really 100%, it comes out at 76% on Coveralls

This is possibly just a matter of adding a `-ignore_re` option to the `cover`
command, but will need to be tested.

**Comment by zmughal** [comment]
@mohawk2 , does this have anything to do with the `-relative_only` flag that you created in your `mohawk2:deepdirs` PR for `Devel::Cover`?

**Comment by mohawk2** [comment]
No, that's only to add a `-r` flag for gcov, which is in C-land. I think the above distro is pure-Perl.

**Comment by zmughal** [comment]
I believe so, but my fix is not ready to go yet.
